google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugProcess.java

```diff
@@ -261,9 +261,12 @@ public void run(@NotNull ProgressIndicator indicator) {
     // Start breakpoints refresh job on first use.
     getStateController().addListener(this);
     getStateController().startBackgroundListening();
-    SwingUtilities.invokeLater(new Runnable() {
-      @Override
-      public void run() {
+    SwingUtilities.invokeLater(() -> {
+      // If a breakpoint handler has not been created before, getBreakpointHandler() will attempt to
+      // create one. But if the project is disposed of, this creation will fail because it will try
+      // to access the project's components when creating the PsiManager. This guards against a
+      // failure in this scenario.
+      if (breakpointHandlers != null || !getXDebugSession().getProject().isDisposed()) {
```

Comment:
This is the first I'm seeing this change by JB.

It seems strange to me that they would wait until the project is disposed to process these anyc events.

> We could manually call this somewhere in the test instead, what do you think?
 
Manually call what? 

I agree that it feels wrong to modify anything but the test in this case unless we can be sure there's problem with the non-test logic.